### DISCUSSION TITLE
💬 Join our Discord server for general discussions.

### DISCUSSION THREAD

**Discussion by Bonapara** [twentyhq/twenty#3642] [General]
https://discord.gg/UfGNZJfAG6

**Comment by deezeddd** [comment]
Hi! The invite has expired, kindly share the new link.

**Comment by Bonapara** [comment]
Updated above.

Thanks @deezeddd ;)

**Comment by Vijaykv5** [comment]
@Bonapara This Link you've updated is showing Invalid.

Please update it with earlier _link_ -> [https://discord.gg/cx5n4Jzs57](https://discord.gg/cx5n4Jzs57) :)

**Reply by Bonapara** [reply]
Done thanks!

**Comment by smartlader-de** [comment]
Expired again @Bonapara

**Reply by Bonapara** [reply]
updated - thanks @smartlader-de
